### ISSUE TITLE
Packer tries to clean the plugins that are being installed

### ISSUE THREAD

**Issue by NTBBloodbath** [wbthomason/packer.nvim#652]
Hey, I'm actually trying to do a hardcore automation in my Neovim setup like install / uninstall plugins on the fly without having to restart Neovim and such stuff and I'm facing a weird bug where packer tries to clean the plugins that are being installed.

- `nvim --version`: v0.5.1
- `git --version`: 2.33.0
- Operating system/version: Fedora Linux 34
- Terminal name/version: kitty

### Steps to reproduce

- Run `PackerClean` and then `PackerInstall` somewhere in your config
- Add a new plugin in your packer setup
- Relaunch Neovim

### Actual behaviour

Packer clean window is raised asking for removal of the plugin that is being installed right now with PackerInstall

### Expected behaviour

Packer should just install the plugin and PackerClean should do nothing because the plugin is not installed yet nor disabled or removed from setup.

### packer files

Basically the only needed things to reproduce this issue would be adding these two lines somewhere in your Neovim configuration

```lua
vim.cmd("PackerClean")
vim.cmd("PackerInstall")
```

Note: I also tried scheduling the `PackerInstall` run so it shouldn't have collisions with `PackerClean` call and vice versa but the weird behavior is still here _sometimes_.

Please let me know if you need extra information or if you want me to try something, I'll be glad to help :)

**Comment by wbthomason** [comment]
Thanks for your report! I'll note that (1) `packer.install()` and `packer.clean()` can be used as functions if you're writing Lua anyway, and (2) `packer.install` _should_ run a `clean` operation anyway. Do you still see the problem if you remove the explicit call to clean? I suspect that this is something to do with the install running in parallel with the clean, whereas the install on its own should do both parts for you in the proper sequence.

**Comment by NTBBloodbath** [comment]
So yeah it does work and I don't see a duplicated clean window anymore. However if I just want to remove commented, removed or disabled plugins on the fly then PackerInstall doesn't do it so I would need a clean call anyway, any idea about how to "bypass" this behavior? :thinking:
